AndroidManifest.xml

```diff
@@ -9,6 +9,28 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+
+    <!-- to start monitor services during boot -->
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
+
+    <!-- to modify Seadroid's own account sync settings -->
+    <uses-permission android:name="android.permission.READ_SYNC_SETTINGS"/>
+    <uses-permission android:name="android.permission.WRITE_SYNC_SETTINGS"/>
+
+    <!-- get list of Seadroid accounts -->
+    <uses-permission android:name="android.permission.GET_ACCOUNTS"
+        android:maxSdkVersion="22"/>
```

Comment:
No risk. Starting with API 23, those permissions are no nonger needed. See e.g.

https://developer.android.com/reference/android/accounts/AccountManager.html#getAuthToken%28android.accounts.Account,%20java.lang.String,%20android.os.Bundle,%20android.app.Activity,%20android.accounts.AccountManagerCallback%3Candroid.os.Bundle%3E,%20android.os.Handler%29

Requesting these permissions on API 23 (Android 6.0) would do no harm. I simply followed the principle of requesting as very permissions as absolutely necessary.